src/test/resources/features/15-attachment/attachements.feature

```diff
@@ -0,0 +1,20 @@
+ Feature: Attachments Management
+
+   Background:
+     Given a user logs in, searches a patient and visits attachment page 
+     When a user clicks on attachment link
+     Then the system loads the attachment page
+
+   #User uploading attachment
+   Scenario: Uploading attachment
+     When a user attaches a file
+     And a user enters a caption
+     And a user clicks Upload file functionality
+     Then the system uploads the attached file
+     
+   #System clearing the forms
+   Scenario: Clearing forms
+     When a user clicks clear forms button
+     Then the system clears the forms
+     And quit browser
```

Comment:
@kdaud cross check against existing ones to ensure your steps are all unique

Reply:
> @kdaud cross check against existing ones to ensure your steps are all unique

Hey @kaweesi I would really love to work on the recommendation but have not yet figured out its implication